packages/client/src/Feed.js

```diff
@@ -25,7 +25,7 @@ const Feed = () => {
   useEffect(() => {
     getAdminStatus(user.userId);
     populateFeed(isAdmin);
-  }, [isAdmin]);
+  }, [isAdmin]); // eslint-disable-line
```

Comment:
This is yelling because user isn't in the dependency thing? Can we add the exact error to this for notation please:
```
eslint-disable-line react-exhaustive-deps
```